docs/modules/ROOT/pages/how-tos/upgrade-v1-to-v2.adoc

```diff
@@ -0,0 +1,32 @@
+= Upgrade from v1 to v2
+
+With the recent update of Kapitan, Helm 3 charts can be used. Therefor the need for the custom lvmd daemonset is gone.
+Therefor a minor change in the `deviceclasses` parameter is necessary.
+
+== Update component parameters
+
+Change `deviceclasses` from `dictionary` to `array`
+
+[source,bash]
+----
+parameters:
+  topolvm:
+    deviceclasses:
+      ssd: <1>
+        volume-group: vgssd
+        default: true
+        spare-gb: 10
+----
+<1> Change this line to `- name: ssd`.
+
+Example for v2:
+[source,bash]
+----
+parameters:
+  topolvm:
+    deviceclasses:
+      - name: ssd
+        volume-group: vgssd
+        default: true
+        spare-gb: 10
+----
```

Comment:
I think the storageclass part also needs a similar change. https://github.com/topolvm/topolvm/blob/main/charts/topolvm/values.yaml#L430

Reply:
See my comment above